website/client/js/services/achievementServices.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+/**
+ * Services that handle achievement logic.
+ */
+
+angular.module('habitrpg').factory('Achievement',
+['$rootScope', function($rootScope) {
+
+  function displayAchievement(achievementName) {
+    $rootScope.openModal('achievements/' + achievementName,
+      {controller:'UserCtrl', size:'sm'});
+  }
+
+  function displayBulkyAchievement(achievementName) {
+    $rootScope.openModal('achievements/' + achievementName,
+      {controller:'UserCtrl'});
```

Comment:
That doesn't work though because the alternative to 'sm' is just to NOT specify a size in the `openModal` call at all.